### PR TITLE
resolved #1768: Update NOTES - CHANGE REPORTED.vbs

### LINKED ISSUE #1768: Remove SSN field from baby born in Change Reported
In DAIL - NEW HIRE, we are x'ing out the client's SSN. In reviewing the baby born section in NOTES >> CHANGE REPORTED, I found that the dialog includes a field for the child's SSN and that it's a mandatory field. I move that the field be removed entirely.

Thoughts...?

### PR DESCRIPTION
**Pull request by StormageddonDLOA**
Removes Baby's SSN from the dialog bog and case notes on the Baby Born part of the script.

### CODE REVIEW

File: Script Files/NOTES/NOTES - CHANGE REPORTED.vbs [hunk 1]
```diff
@@ -108,7 +106,7 @@ BeginDialog HHLD_Comp_Change_Dialog, 0, 0, 291, 175, "Household Comp Change"
   EditBox 95, 50, 60, 15, date_reported
   Text 165, 55, 45, 10, "Effective Date"
   EditBox 215, 50, 70, 15, effective_date
-  CheckBox 110, 70, 100, 10, "Is the change temporary?", temporary_change_checkbox
+  CheckBox 110, 70, 110, 10, "Check if the change is temporary.", temporary_change_checkbo1
```
Comment (theVKC) [inline Script Files/NOTES/NOTES - CHANGE REPORTED.vbs]: This says "checkbo1". Guessing that isn't right... did you test this version and try the new checkbox?

**Comment by theVKC** [comment]
See my note @StormageddonDLOA. Thanks!

**Comment by StormageddonDLOA** [comment]
Yes I did test it and you are correct it should say "checkbox" not "checkbo1". Somehow it still worked but the "1" is incorrect.

**Comment by theVKC** [comment]
Hmmm. If you tested this and it still did that, that's very strange. Did you copy the code into GitHub on the web? Maybe something goofy happened in the clipboard? I don't know.

In any case, can you update your branch to reflect the correct variable? Thanks!

**Comment by StormageddonDLOA** [comment]
Fixed.

**Pull request merged**